### ISSUE TITLE
LocationSettings: WakeLockTime has no effect

### ISSUE THREAD

**Issue by komape** [rekab-app/background_locator#96]
The parameter `wakeLockTime` for `LocationSettings` has no effect. It always stops getting location updates 30 seconds after the app goes into background.

Flutter version: 1.17.3
Device: Pixel 3a API 29 (android-x86 emulator)
background_locator version: 1.1.5+1

**Comment by gpibarra** [comment]
Have you tried on a physical computer (without emulator)? In emulator

**Comment by komape** [comment]
I tried it out with two different real phones. No problems. Seems like it's a problem of the emulator.

**Comment by mehdok** [comment]
@komape Can you see the notification after 30 second? `wakeLockTime` is just for preventing device from deep sleep;

**Comment by komape** [comment]
Do you mean the symbol in the upper status bar? 

I have to check but as far as I remember, the symbol was still there.

**Comment by mehdok** [comment]
> I tried it out with two different real phones. No problems. Seems like it's a problem of the emulator.

Somehow I managed to skip this comment. It seems that your problem is fixed, is that right?


> I have to check but as far as I remember, the symbol was still there.

As far as you see the notification the service is running;

**Comment by komape** [comment]
Yes, the problem only exists in the emulator. Had no problems with real devices so far.
